src/tokenlists/balancer/overwrites.ts

```diff
@@ -986,4 +986,11 @@ export const overwrites: OverwritesForList = {
       },
     },
   },
+  [Network.Mode]: {
+    '0xD08a2917653d4E460893203471f0000826fb4034': {
+      extensions: {
+        coingeckoId: 'balancer',
+      },
```

Comment:
I've never seen this schema before? Is it part of the tokenlist standard or something custom?

Reply:
thanks a way to link pricing between the same tokens when coingecko doesn't do that on it's own yet.